webapp/src/main/java/com/box/l10n/mojito/service/tm/TranslatorWithInheritance.java

```diff
@@ -89,7 +92,13 @@ public String getTranslationFromTextUnitDTO(TextUnitDTO textUnitDTO, String sour
             translation = textUnitDTO.getTarget();
         } else if (InheritanceMode.USE_PARENT.equals(inheritanceMode)) {
             logger.debug("No TextUnitDTO, fallback to source");
+
             translation = source;
+
+            if(!Strings.isNullOrEmpty(untranslatedStringsFormat))
```

Comment:
wondering if we should have the parameter/command name being more tight to "USE_PARENT" like use `useParentUntranslatedPattern`?

If we have a more independent param (which might be fine), that logic should be done regardless of the inheritance mode, I guess. It would make little sense with the `REMOVE_UNTRANSLATED` though and we don't have any other  mode for now. Or we could also have a special inheritance mode for it?

Leaning toward just having that param as a config of USE_PARENT like your current code but renaming.

Knit: I'd change the if statement to have the else close do `translation = source` 

Thoughts?